test/integration/rules/aria-required-children/aria-required-children.html

```diff
@@ -55,3 +55,12 @@
     <li role="menuitem" id="ignore14">>Trash</li>
   </ul>
 </div>
+
+<div role="suggestion" id="pass8">
+  <span role="deletion">option</span>
+  <span role="insertion">option</span>
+</div>
+
+<div role="suggestion" id="fail8">
```

Comment:
Good start. So with our integration tests we want to be really thorough and test all possible states, so we'll want 3 more tests added. One for `suggestion` without any children, and one for `suggestion` with just an `insertion` child, and one for `suggestion` with just a `deletion` child.

Reply:
Unable to test for one child due to `requiredOwned` (as discussed). However [ARIA Suggestion Docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Suggestion_role) say that both need to be present, playground throws no errors if using one child